cs-vmware/src/main/java/io/cloudslang/content/vmware/services/ClusterComputeResourceService.java

```diff
@@ -89,6 +94,27 @@
         return resultMap;
     }
 
+    public String getVmOverride(final HttpInputs httpInputs, final VmInputs vmInputs) throws Exception {
+        final ConnectionResources connectionResources = new ConnectionResources(httpInputs, vmInputs);
+
+        final ManagedObjectReference clusterMor = new MorObjectHandler().getSpecificMor(connectionResources, connectionResources.getMorRootFolder(),
+                ClusterParameter.CLUSTER_COMPUTE_RESOURCE.getValue(), vmInputs.getClusterName());
+
+        final ClusterConfigInfoEx clusterConfigInfoEx = getClusterConfiguration(connectionResources, clusterMor, vmInputs.getClusterName());
+
+        final String restartPriority;
+        if(isNotBlank(vmInputs.getVirtualMachineId()) || isNotBlank(vmInputs.getVirtualMachineName())) {
```

Comment:
you either don't import 'isNotBlank' statically or import 'StringUtilities.isNotEmpty' (and other StringUtilities methods used in the class) statically, we should keep the class consistent